src/metrickit.cs

```diff
@@ -141,21 +141,22 @@ interface MXMetaData : NSSecureCoding {
 		[Export ("JSONRepresentation")]
 		NSData JsonRepresentation { get; }
 
-#if !MONOMAC
+		[NoMac]
 		[Internal]
 		[Deprecated (PlatformName.iOS, 14,0)]
 		[Export ("DictionaryRepresentation")]
 		NSDictionary _DictionaryRepresentation13 { get; }
 
+		[NoMac]
 		[Internal]
 		[iOS (14,0)]
 		[MacCatalyst (14,0)]
 		[Export ("dictionaryRepresentation")]
 		NSDictionary _DictionaryRepresentation14 { get; }
-#else 
+
+		[NoiOS][NoMacCatalyst][NoWatch][NoTV]
 		[Export ("dictionaryRepresentation")]
 		NSDictionary DictionaryRepresentation { get; }
```

Comment:
I think it would be more correct to merge the two `dictionaryRepresentation` properties, to have the same availability attributes on all platforms:

```cs
 		[iOS (14,0)]
 		[MacCatalyst (14,0)]
 		[Export ("dictionaryRepresentation")]
#if MONOMAC
 		NSDictionary DictionaryRepresentation { get; }
#else
 		[Internal]
 		NSDictionary _DictionaryRepresentation14 { get; }
#endif

Reply:
Done.